spaceMissionManager-service/src/main/java/cz/muni/fi/services/impl/ServiceDataAccessException.java

```diff
@@ -0,0 +1,17 @@
+package cz.muni.fi.services.impl;
+
+import org.springframework.dao.DataAccessException;
+
+/**
+ * Thrown when there is a problem with accessing the data on the lower layer.
+ * @author Vojtech Bruza
+ */
+public class ServiceDataAccessException extends DataAccessException {
+    public ServiceDataAccessException(String msg) {
```

Comment:
I suggest adding logger to log the given message

Reply:
Any suggestions to which logger shall I use, please?

Reply:
import java.util.logging.Logger;
private Logger logger = Logger.getLogger(MissionServiceImpl.class.getName());